frontend/components/syncing/Vercel/CreateVercelSync.tsx

```diff
@@ -0,0 +1,343 @@
+import GetVercelProjects from '@/graphql/queries/syncing/vercel/getProject.gql'
+import GetAppSyncStatus from '@/graphql/queries/syncing/getAppSyncStatus.gql'
+import GetAppEnvironments from '@/graphql/queries/secrets/getAppEnvironments.gql'
+import GetSavedCredentials from '@/graphql/queries/syncing/getSavedCredentials.gql'
+import CreateNewVercelSync from '@/graphql/mutations/syncing/vercel/createVercelSync.gql'
+import { useLazyQuery, useMutation, useQuery } from '@apollo/client'
+import { Fragment, useContext, useEffect, useState } from 'react'
+import { Button } from '../../common/Button'
+import { 
+  EnvironmentType, 
+  ProviderCredentialsType,
+  VercelProjectType 
+} from '@/apollo/graphql'
+import { Combobox, RadioGroup, Transition } from '@headlessui/react'
+import clsx from 'clsx'
+import { FaAngleDoubleDown, FaChevronDown, FaCircle, FaDotCircle } from 'react-icons/fa'
+import { toast } from 'react-toastify'
+import { TbBrandVercel } from 'react-icons/tb'
```

Comment:
```suggestion
import { SiVercel } from 'react-icons/si'
```

Lets stick with the icon from Simple Icons for consistency